php/php.editor/test/unit/data/goldenfiles/org/netbeans/modules/php/editor/csl/NavigatorTest/structure/anonymousClassInClassScope.pass

```diff
@@ -4,7 +4,7 @@
 |--test1 [130, 1059] : ESCAPED{test1}ESCAPED{(}ESCAPED{)}
 |--test2 [1081, 1256] : ESCAPED{test2}ESCAPED{(}ESCAPED{)}
 |--test3 [1278, 1510] : ESCAPED{test3}ESCAPED{(}ESCAPED{)}
-|-#anon#anonymousClassInClassScope_php#1 [155, 1052] : ESCAPED{{}}<font color="#999999">::ESCAPED{ClassScopeTest}</font><font color="#999999">:ESCAPED{ClassScopeTestInterface}</font><font color="#999999">#ESCAPED{ClassScopeTestTrait}</font>
+|-#anon#anonymousClassInClassScope_php#1 [159, 1052] : ESCAPED{{}}<font color="#999999">::ESCAPED{ClassScopeTest}</font><font color="#999999">:ESCAPED{ClassScopeTestInterface}</font><font color="#999999">#ESCAPED{ClassScopeTestTrait}</font>
```

Comment:
`new class () {};`
Changed start position from `new` to `class`.
By this change:
https://github.com/apache/netbeans/blob/9a45ad273f06d392ce724b92966dff6b7e73cbb8/php/php.editor/src/org/netbeans/modules/php/editor/model/nodes/ClassInstanceCreationInfo.java#L56-L65